CHANGELOG.md

```diff
@@ -1,3 +1,60 @@
+   - [parsec-se-driver] Adds a recipe to build [Parsec Secure Element driver](https://github.com/parallaxsecond/parsec-se-driver) 0.4.0, which is a dependency of Edge Core and mbed-fcce package when compiled with `MBED_EDGE_CORE_CONFIG_PARSEC_TPM_SE_SUPPORT=ON`.
+   - Adds `meta-rust`, `meta-clang` and `meta-security/meta-tpm`, which are prerequisites to build Parsec and related packages.
+   - [parsec-tool] Adds a recipe to build [Parsec Tool](https://github.com/parallaxsecond/parsec-tool), a command-line utility to debug and cross-validate the working of Parsec service and TPM.
+- [edge-core] Updates Edge Core to [0.16.1](https://github.com/PelionIoT/mbed-edge/blob/master/CHANGELOG.md#release-0160-2021-3-15).
+   - Reduces the default log level to WARN.
+   - Adds `mbed_cloud_client_user_config.h`, so you can set the values for your use case. This overwrites the default config options set by Edge Core. The default lifetime value is set to 1800s (30min).
+   - Explicitly defines HTTP_PROXY and HTTPS_PROXY environment variables.
+- [edge-examples] Updates examples to [0.16.0](https://github.com/PelionIoT/mbed-edge-examples/blob/master/CHANGELOG.md#release-0160-2021-03-15).
+- [mbed-fcce] Upgrades factory-configurator-client-example to v4.7.1.
+   - Renames the package name from `mbed-fcc` to `mbed-fcce`.
+   - Explicitly defines HTTP_PROXY and HTTPS_PROXY environment variables.
+- [verified-logging] By default, the gateway is configured with persistent journal logging for LMP UltraZed-EG IOCC and i.MX 8M Mini EVK. To disable persistent logging, set flag `VOLATILE_LOG_DIR = "no"` in `local.conf`, and update the `Storage` in recipes-core/systemd/systemd-conf/journald.conf. Note: If you disable persistent logging, the FSS feature won't work.
+- Updates `identity-tool`, `kubelet` and `info-tool` package source file protocol from SSH to HTTPS.
+
+### Bug fixes
+
+- [pt-example] Pelion Edge 2.2 used protocol translator example 0.13.0, which wasn't compatible with Edge Core 0.15.0. We fixed this by upgrading the example to version 0.16.0.
+- In Pelion Edge 2.2, using the i.MX 8M Mini EVK in production mode with firmware update enabled failed with a FOTA_ASSERT after the reboot. This has been fixed.
+
+### Known issues
+
+- The Pelion Device Management portal is not correctly updated after a firmware campaign in some instances.
+- [maestro] The FeatureMgmt config resource is initialized with a maximum 3.8KB of file content. The remaining file content is truncated during initialization. This is most likely due to the limitation of the gorilla/websocket library but needs further investigation. However, you can still push a file size of a maximum of 64KB through cloud service APIs.
+- [pt-example] `cpu-temperature` device reports random values because the default CPU temperature file is not the same on Yocto and LmP.
```

Comment:
Are these values truly random or should we replace "random" with "incorrect"?

Reply:
Yes, they are random.